_data/documentsupdated.yaml

```diff
@@ -369,29 +369,47 @@
     - documentname: Third-Party Bots
       pages:
         - pagename: Getting Started
-        - pagename: Bot Status Dashboard
-        - pagename: Bot Conversation Errors
+        - pagename: Bot Types
         - pagename: Google Dialogflow
         - pagename: Google Dialogflow ES
           subpages:
             - subpagename: Introduction
             - subpagename: Basic Content
             - subpagename: Advanced Features
         - pagename: Google Dialogflow CX
+          subpages:
+            - subpagename: Introduction
+            - subpagename: Basic Content
+            - subpagename: Advanced Features
         - pagename: IBM Watson Assistant
-        - pagename: IBM Watson Assistant Version 2
+          subpages:
+            - subpagename: Introduction
+            - subpagename: Basic Content
+            - subpagename: Advanced Features
         - pagename: Amazon Lex
+          subpages:
+            - subpagename: Introduction
+            - subpagename: Basic Content
+            - subpagename: Advanced Features        
         - pagename: Microsoft Direct Line
           subpages:
             - subpagename: Introduction
             - subpagename: Basic Content
             - subpagename: Advanced Features
             - subpagename: PVA
-        - pagename: Custom Integration
-        - pagename: Hook Configuration
+        - pagename: LivePerson Functions
```

Comment:
Should this be called LivePerson Functions already? Its not part of the current release

Reply:
This is not going to be deployed/merged after this release